src/client/java/teammates/client/scripts/sql/SeedDb.java

```diff
@@ -157,7 +157,7 @@ protected void persistAdditionalData() {
                 String accountRequestEmail = String.format("Account Email %s", i);
                 String accountRequestInstitute = String.format("Account Institute %s", i);
                 AccountRequest accountRequest = AccountRequestAttributes
-                        .builder(accountRequestName, accountRequestEmail, accountRequestInstitute)
+                        .builder(accountRequestEmail, accountRequestName, accountRequestInstitute)
```

Comment:
The method is `builder(String email, String institute, String name)`, so need to change the institute placing too

Reply:
Good catch have amended this